### PR TITLE
docs(common): Document NuGet API keys

### PR DESCRIPTION
**Pull request by dimodi**

### CODE REVIEW

File: installation/nuget.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Telerik Private NuGet Feed
 page_title: Telerik NuGet Feed
-description: How to use the Telerik Private NuGet Feed.
+description: How to add and use the Telerik private NuGet source.
```
Comment (yordan-mitev) [inline installation/nuget.md:4]: Try adding more detailed descriptions, they must contain around 150 symbols.
```suggestion
description: Explore the different approaches of adding the Telerik private NuGet source to your system, and start using NuGet packages to install the Telerik Blazor components.
```

File: installation/nuget.md [hunk 2]
```diff
@@ -10,122 +10,129 @@ position: 1
 
 # Telerik Private NuGet Feed
 
-This article explains how to add the private Telerik NuGet feed to your system. You can use it to obtain the Telerik UI for Blazor components.
+This article explains how to add the private Telerik NuGet package source to your environment. You can use it to obtain the Telerik UI for Blazor components instead of [setting up a local NuGet feed]({%slug installation/zip%}#set-up-a-local-nuget-feed-in-visual-studio).
 
-@[template](/_contentTemplates/common/get-started.md#nuget-update-note)
+There are several ways to setup the remote Telerik NuGet feed:
```
Comment (yordan-mitev) [inline installation/nuget.md:15]:
```suggestion
You can set up the remote Telerik NuGet feed in the following ways:
```

File: installation/nuget.md [hunk 3]
```diff
@@ -10,122 +10,129 @@ position: 1
...
 
-#### There are several approaches to set up the Telerik NuGet feed
+* [Use Visual Studio](#use-visual-studio)
+* [Use the .NET CLI](#use-the-net-cli)
+* [Edit the Nuget.Config file](#edit-the-nugetconfig-file)
 
-* [Visual Studio GUI](#visual-studio-gui)
+>tip When working with the .NET CLI or editing the `NuGet.Config` manually, you can use your Telerik account credentials or a [NuGet API Key](#use-nuget-api-key). If you are logging at telerik.com through single sign on (SSO), use a [NuGet API Key](#use-nuget-api-key).
```
Comment (yordan-mitev) [inline installation/nuget.md:21]:
```suggestion
>tip When working with the .NET CLI or editing the `NuGet.Config` manually, you can use your Telerik account credentials or a [NuGet API Key](#use-nuget-api-key). If you are logging in to telerik.com through single sign-on (SSO), use a [NuGet API Key](#use-nuget-api-key).
```

File: installation/nuget.md [hunk 4]
```diff
@@ -10,122 +10,129 @@ position: 1
...
+When adding NuGet sources in Visual Studio, the credentials are encrypted and stored outside the `NuGet.Config` file.
 
->caption Here are the steps to add the Telerik NuGet Feed through Visual Studio
+Refer to the [Microsoft documentation about using packages in Visual Studio](https://learn.microsoft.com/en-us/nuget/consume-packages/install-use-packages-visual-studio), or follow the steps below for Visual Studio on Windows.
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/dJo1Ij4CcIY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+1. Open Visual Studio and go to Tools > NuGet Package Manager > Package Manager Settings > Package Sources.
 
-1. Clear the local NuGet caches to ensure no cached Telerik packages exist (it is important in the steps later where your credentials will be stored by Windows). You can run [`dotnet nuget locals all --clear`](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-nuget-locals) in your CLI.
+1. Click the `[+]` button at the top right hand side.
```
Comment (yordan-mitev) [inline installation/nuget.md:36]: Buttons and other UI elements must be **bold**.
```suggestion
1. Click the **+** button at the top right-hand side.
```

File: installation/nuget.md [hunk 5]
```diff
@@ -10,122 +10,129 @@ position: 1
...
+1. Open a project that references a Telerik NuGet package. For example, generate one through our [New Project Wizard]({%slug getting-started-vs-integration-new-project%}).
+    * Make sure to remove local `NuGet.Config` files from the solution that contain information about Telerik packages. 
     
-1. From the **Build** menu select **Rebuild Solution**.
+1. Rebuild the solution.
 
-1. There will be a Windows prompt asking for credentials for the Telerik feed. Enter the **email** and **password** you use to log in to telerik.com.
-    * Make sure to **select** the **Remember my password** checkbox.
+1. A Windows prompt will ask for the Telerik feed credentials. Enter your Telerik email and password.
+    * Check "Remember my password".
```
Comment (yordan-mitev) [inline installation/nuget.md:48]:
```suggestion
    * Check **Remember my password**.
```

**Review by yordan-mitev — APPROVED** [review]

File: installation/nuget.md [hunk 6]
```diff
@@ -10,122 +10,129 @@ position: 1
...
+1. Open a project that references a Telerik NuGet package. For example, generate one through our [New Project Wizard]({%slug getting-started-vs-integration-new-project%}).
+    * Make sure to remove local `NuGet.Config` files from the solution that contain information about Telerik packages. 
     
-1. From the **Build** menu select **Rebuild Solution**.
+1. Rebuild the solution.
 
-1. There will be a Windows prompt asking for credentials for the Telerik feed. Enter the **email** and **password** you use to log in to telerik.com.
-    * Make sure to **select** the **Remember my password** checkbox.
+1. A Windows prompt will ask for the Telerik feed credentials. Enter your Telerik email and password.
+    * Check **Remember my password**.
```
Comment (dimodi) [inline installation/nuget.md:48]:
```suggestion
    * Check the Remember My Password checkbox.
```

**Pull request merged**